packages/babel-types/src/definitions/flow.js

```diff
@@ -1,136 +1,156 @@
 // @flow
 import defineType, {
-  assertEach,
-  assertNodeType,
+  arrayOfType,
+  assertOneOf,
   assertValueType,
-  chain,
+  validate,
+  validateOptional,
+  validateOptionalType,
+  validateType,
 } from "./utils";
 
 defineType("AnyTypeAnnotation", {
   aliases: ["Flow", "FlowBaseAnnotation"],
-  fields: {
-    // todo
-  },
 });
 
 defineType("ArrayTypeAnnotation", {
   visitor: ["elementType"],
   aliases: ["Flow"],
   fields: {
-    // todo
+    elementType: validateType("Flow"),
```

Comment:
This whould also allow thing like `TypeAlias` and `ClassImplements`, but it should only allow types.

Reply:
What would you use instead? `FlowBaseAnnotation`?